app/renderer/actions/Session.js

```diff
@@ -310,6 +310,38 @@ export function newSession (caps, attachSessId = null) {
         }
         https = session.server.browserstack.ssl = (parseInt(port, 10) === 443);
         break;
+      case ServerTypes.lambdatest:
+        host = session.server.lambdatest.hostname = process.env.LAMBDATEST_HOST || 'mobile-hub.lambdatest.com';
+        port = session.server.lambdatest.port = process.env.LAMBDATEST_PORT || 443;
+        path = session.server.lambdatest.path = '/wd/hub';
+        const isProxyChecked = session.server.advanced.useProxy;
+        username = session.server.lambdatest.username || process.env.LAMBDATEST_USERNAME;
+        if (desiredCapabilities.hasOwnProperty('lt:options')) {
+          desiredCapabilities['lt:options'].source = 'appiumdesktop';
+          desiredCapabilities['lt:options'].isRealMobile = true;
+          if (isProxyChecked) {
+            if(session.server.advanced.proxy == undefined) desiredCapabilities['lt:options'].proxyUrl = '';
+            else desiredCapabilities['lt:options'].proxyUrl = `${session.server.advanced.proxy}`;
+          }
+        } else {
+          desiredCapabilities['lambdatest:source'] = 'appiumdesktop';
+          desiredCapabilities['lambdatest:isRealMobile'] = true;
+          if (isProxyChecked) {
+            if(session.server.advanced.proxy == undefined) desiredCapabilities['lambdatest:proxyUrl'] = '';
```

Comment:
`==` should be `===` in JS

Reply:
this is weird, we did copy the package-lock from main branch and even I can not find any difference in the file.

Reply:
Then, could you create a new PR with the necessary changes?
I guess GitHub diff messed by some reason.